Python Quiz Game

```diff
@@ -0,0 +1,57 @@
+import random
+
+
+		print("Welcome to Q-Quiz!\nGood luck")
+
+
+		answer=input("Ready?")
+		score=0
+
+
+		if answer.lower()=='yes':
+		    print("Great!")
+		else:
+		    print("See you later!")
+
+
+		QUESTIONS = [
+		    ('#question1', '#answer1'),
+		    ('#question2', '#answer2'),
+		    ('#question3', '#answer3'),
+		]
+
+
+		num_questions=len(QUESTIONS)
+		questions = random.sample(list(QUESTIONS), k = num_questions)
+		num = random.randint(0,#n)
+		CORRECT = [
+		    "#output1",
+		    "#output2",
+		    "#output3",
+		]
+
+
+		INCORRECT = [
+		    "#output1",
+		    "#output2",
+		    "#output3",
+		]
+
+
+		correct_answer = CORRECT[num]
+		incorrect_answer = INCORRECT[num]
+
+
+		for question, right_answer in questions:
+		    answer = input(f"{question}")
+		    if answer.lower() == right_answer:
+		        score += 1
+		        print(correct_answer)
+		    else:
+		        print(incorrect_answer)
+
+
+		print("⭐ Congratulations! You've made it! ⭐\nYou answered",score,"questions correctly!")
+		mark=(score/total_questions)*100
+		print('Total marks:',mark)
+		print('Thank you for playing, I hope you had fun!')
```

Comment:
bad indentation

Reply:
I wrote it locally and copied it over, that's why. Sorted now, thank you for pointing it out I didn't notice.

Reply:
suggestion outdated, conversation marked resolved